includes/rplus-wp-team-list.php

```diff
@@ -10,8 +10,9 @@
  * @package WP_Team_List
  */
 
-$role        = wp_team_list()->get_user_role( $user );
-$description = get_user_meta( $user->ID, 'description', true );
+$role              = wp_team_list()->get_user_role_name( $user );
+$role_display_name = wp_team_list()->get_user_role( $user );
```

Comment:
It's a bit confusing at first that `$role_display_name !== wp_team_list()->get_user_role_name()`.